### PR TITLE
Update all hardcoded TextStyle. All text style should relate on AppTheme, or use / extend common style

### PR DESCRIPTION
**Pull request by sudoosama**
┆Issue is synchronized with this [Jira Task](https://highvibe.atlassian.net/browse/HVN1-255) by [Unito](https://www.unito.io/learn-more)

### CODE REVIEW

**Review by AlexVegner — CHANGES_REQUESTED** [review]
Please make shure that you fixed all placess with hardcoded `TextStyle(` and `Color(0x`

File: mobile/lib/modules/playlist/edit_playlist/edit_playlist_page.dart [hunk 1]
```diff
@@ -186,51 +182,49 @@ class _EditPlayListPageState
...
-              ),
-              focusedBorder: UnderlineInputBorder(
-                borderSide: BorderSide(color: Color(0xFF525366), width: 1),
-              ),
-              hintText: PlaylistStrings.enterPlaylistName,
-              hintStyle: TextStyle(color: Colors.white),
-            ),
-            style: const TextStyle(color: Colors.white),
-          ),
+              controller: _titleController,
```
Comment (AlexVegner) [inline mobile/lib/modules/playlist/edit_playlist/edit_playlist_page.dart:193]: Use dart file format for dart formation. 
We have it in vscode workspace configuration. 
- Automatic dart file formating, 
- automatic import optimization.

File: mobile/lib/modules/playlist/edit_playlist/edit_playlist_page.dart [hunk 2]
```diff
@@ -186,51 +182,49 @@ class _EditPlayListPageState
...
-              ),
-              focusedBorder: UnderlineInputBorder(
-                borderSide: BorderSide(color: Color(0xFF525366), width: 1),
-              ),
-              hintText: PlaylistStrings.enterDescription,
-              hintStyle: TextStyle(color: Colors.white),
-            ),
-            style: const TextStyle(color: Colors.white),
-          ),
+              controller: _descriptionController,
```
Comment (AlexVegner) [inline mobile/lib/modules/playlist/edit_playlist/edit_playlist_page.dart:215]: Use dart file format for dart formation.

File: mobile/lib/modules/playlist/widgets/ui_utils.dart [hunk 3]
```diff
@@ -36,14 +36,12 @@ void showSuccessDialog({
   }
 }
 
-Widget snackBarLight({@required String message}) {
+Widget snackBarLight({@required String message, BuildContext context}) {
```
Comment (AlexVegner) [inline mobile/lib/modules/playlist/widgets/ui_utils.dart:39]: BuildContext should be first argument for all widget build methods where we need it.

File: mobile/lib/modules/profile/widgets/profile_input_field.dart [hunk 4]
```diff
@@ -36,19 +35,14 @@ class ProfileInputField extends StatelessWidget {
...
               decoration: InputDecoration(
-                contentPadding: const EdgeInsets.symmetric(vertical: 20),
-                border: const UnderlineInputBorder(
-                    borderSide: BorderSide(color: Colors.white24)),
-                hintText: '$hintText',
-                hintStyle: const TextStyle(
-                  color: Color(0xFF525366),
-                ),
-              ),
+                  contentPadding: const EdgeInsets.symmetric(vertical: 20),
```
Comment (AlexVegner) [inline mobile/lib/modules/profile/widgets/profile_input_field.dart:38]: Use dart file format for dart formation.

File: mobile/lib/modules/wallet/components/transactions/transaction_list.dart [hunk 5]
```diff
@@ -75,17 +75,14 @@ class TransactionList extends StatelessWidget {
...
                 title: Text(
                   '''Tap on a transaction address below, to visit Etherscan for more details''',
-                  style: TextStyle(
-                    fontSize: 14,
-                    color: Colors.grey,
-                  ),
+                  style: AppStyles.normal14Hint(context),
                 ),
-                leading: Icon(Icons.info),
+                leading:const Icon(Icons.info),
```
Comment (AlexVegner) [inline mobile/lib/modules/wallet/components/transactions/transaction_list.dart:85]: Use dart file format for dart formation.
`leading:const` should be `leading: const`

File: mobile/lib/modules/wallet/components/wallet/balance.dart [hunk 6]
```diff
@@ -179,21 +177,19 @@ class Balance extends StatelessWidget {
                         mainAxisAlignment: MainAxisAlignment.start,
                         crossAxisAlignment: CrossAxisAlignment.start,
                         children: <Widget>[
-                          const Padding(
-                            padding: EdgeInsets.fromLTRB(20, 10, 20, 0),
+                          Padding(
+                            padding:const  EdgeInsets.fromLTRB(20, 10, 20, 0),
```
Comment (AlexVegner) [inline mobile/lib/modules/wallet/components/wallet/balance.dart:181]: Use dart file format for dart formation.
`const  EdgeInsets` should be `const EdgeInsets` (one space)

File: mobile/lib/modules/wallet/pages/auth/passcode/passcode_page.dart [hunk 7]
```diff
@@ -122,9 +121,10 @@ class PassCodePageState extends State<PassCodePage> {
           padding: const EdgeInsets.only(bottom: 25),
           child: FlatButton(
             onPressed: onFaceId,
-            child: const Text(
+            child: Text(
               'Use Face ID',
-              style: TextStyle(color: AppColors.textBlue, fontSize: 18),
+style: AppStyles.normal18WithColor(context,
```
Comment (AlexVegner) [inline mobile/lib/modules/wallet/pages/auth/passcode/passcode_page.dart:126]: Use dart file format for dart formation.

File: mobile/lib/modules/wallet/components/wallet/balance.dart [hunk 8]
```diff
@@ -117,21 +117,19 @@ class Balance extends StatelessWidget {
...
-                              style: TextStyle(
-                                  fontSize: 10.0, color: Colors.white),
+                              style: AppStyles.normal10White(context),
                             ),
                           ),
-                          const Center(
+                           Center(
                             child: Padding(
-                              padding: EdgeInsets.fromLTRB(20, 10, 20, 0),
+                              padding:const EdgeInsets.fromLTRB(20, 10, 20, 0),
```
Comment (AlexVegner) [inline mobile/lib/modules/wallet/components/wallet/balance.dart:129]: Use dart file format for dart formation.
`padding:const` should be `padding: const`

File: mobile/lib/widgets/tags/tag_item.dart [hunk 9]
```diff
@@ -69,6 +72,7 @@ class TagItem extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
+    _getTextStyle(context);
```
Comment (AlexVegner) [inline mobile/lib/widgets/tags/tag_item.dart:75]: Epic usage)
You need to use result of _getTextStyle(context);...

**Review by AlexVegner — CHANGES_REQUESTED** [review]

File: mobile/lib/modules/profile/widgets/profile_input_field.dart (hunk 4 above)
Comment (sudoosama) [reply mobile/lib/modules/profile/widgets/profile_input_field.dart:38]: Okay

File: mobile/lib/modules/profile/current_user/create_post/edit_post_page.dart [hunk 10]
```diff
@@ -33,9 +33,9 @@ class EditPostPageState
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: const Text(
+        title: Text(
           Strings.createPost,
-          style: TextStyle(fontWeight: FontWeight.bold),
+          style: AppStyles.normal14White(context),
```
Comment (AlexVegner) [inline mobile/lib/modules/profile/current_user/create_post/edit_post_page.dart:38]: Should be `bold`

File: mobile/lib/modules/meditation/past_meditation.dart [hunk 11]
```diff
@@ -158,19 +144,13 @@ class PastMeditations extends StatelessWidget {
                       label: RichText(
                         text: TextSpan(
                             text: '$exp ',
-                            style: const TextStyle(
-                              fontSize: 16.0,
-                              fontWeight: FontWeight.bold,
-                              color: Color(0xFF525366),
-                            ),
+                            style: AppStyles.normal16WithColor(context,
```
Comment (AlexVegner) [inline mobile/lib/modules/meditation/past_meditation.dart:147]: Should be `bold`

File: mobile/lib/modules/live/components/widgets/add_description_widget.dart [hunk 12]
```diff
@@ -65,18 +66,16 @@ class _InputTextField extends StatelessWidget {
   _InputTextField(this.controller, this.hintText);
   @override
   Widget build(BuildContext context) {
-    const fontSize = 16.0;
     const textColor = Color.fromRGBO(255, 255, 255, 0.5);
     return TextField(
-      style: const TextStyle(color: Colors.white, fontSize: fontSize),
+      style: AppStyles.normal16White(context),
```
Comment (AlexVegner) [inline mobile/lib/modules/live/components/widgets/add_description_widget.dart:71]: Use `fontSize` parameter

File: mobile/lib/modules/discover/videos/video_card.dart [hunk 13]
```diff
@@ -64,7 +64,7 @@ class VideoCard extends StatelessWidget {
             child: Center(
               child: CircleAvatar(
                 radius: 20,
-                backgroundColor: Color(0x40000222),
+                backgroundColor: AppColors.backgroundColor,
```
Comment (AlexVegner) [inline mobile/lib/modules/discover/videos/video_card.dart:67]: it should be background color with transparency 0x40

File: mobile/lib/modules/discover/live_now/live_card.dart [hunk 14]
```diff
@@ -78,7 +78,7 @@ class LiveCard extends StatelessWidget {
                   child: Center(
                     child: CircleAvatar(
                       radius: 40,
-                      backgroundColor: Color(0x40000222),
+                      backgroundColor: AppColors.backgroundColor,
```
Comment (AlexVegner) [inline mobile/lib/modules/discover/live_now/live_card.dart:81]: It should be backgroundColor with transparency 0x40

File: mobile/lib/modules/discover/audios/audio_card.dart [hunk 15]
```diff
@@ -86,7 +86,7 @@ class AudioCard extends StatelessWidget {
           Container(
             width: 80,
             height: 90,
-            color: const Color(0x50000000),
+            color: Colors.black,
```
Comment (AlexVegner) [inline mobile/lib/modules/discover/audios/audio_card.dart:89]: it should be black with transparency 0x50

File: core/lib/common/themes/app_styles.dart [hunk 16]
```diff
@@ -4,6 +4,34 @@ import 'package:flutter/material.dart';
...
   //TODO all styles shold depends for Theme.of(context).xxx
 
+  static TextStyle bold60PlayfairWhite(BuildContext context) =>
+      Theme.of(context).textTheme.headline2.copyWith(
+            color: Colors.white,
+          );
+  static TextStyle bold60PlayfairWithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.headline2.copyWith(
+            fontSize: 15,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:14]: fontSize: 15, ???

File: core/lib/common/themes/app_styles.dart [hunk 17]
```diff
@@ -4,6 +4,34 @@ import 'package:flutter/material.dart';
...
+          );
+
+  static TextStyle bold48PlayfairWhite(BuildContext context) =>
+      Theme.of(context).textTheme.headline3.copyWith(
+            color: Colors.white,
+          );
+
+  static TextStyle normal42White(BuildContext context) =>
+      Theme.of(context).textTheme.headline4.copyWith(
+            fontSize: 42,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:25]: It should have normal weight not bold

File: core/lib/common/themes/app_styles.dart [hunk 18]
```diff
@@ -20,6 +48,12 @@ abstract class AppStyles {
             fontSize: 24,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:48]: bold20PlayfairWhite should have fontSize: 20 not 24

File: core/lib/common/themes/app_styles.dart [hunk 19]
```diff
@@ -61,6 +101,19 @@ abstract class AppStyles {
...
+      Theme.of(context).textTheme.subtitle1.copyWith(
+            fontSize: 15,
+            color: Colors.white,
+          );
+
+  static TextStyle normal15WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.button.copyWith(
+            fontSize: 15,
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:114]: You should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 20]
```diff
@@ -93,6 +150,28 @@ abstract class AppStyles {
       Theme.of(context).textTheme.button.copyWith(
             color: Colors.white,
           );
+  static TextStyle normal14ColorwithHeight(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.button.copyWith(
+            height: 1.3,
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:157]: Uou should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 21]
```diff
@@ -93,6 +150,28 @@ abstract class AppStyles {
...
+  static TextStyle bold14White(BuildContext context) =>
+      Theme.of(context).textTheme.button.copyWith(
+            fontWeight: FontWeight.w600,
+            color: Colors.white,
+          );
+  static TextStyle bold14WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.button.copyWith(
+            fontWeight: FontWeight.bold,
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:168]: You should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 22]
```diff
@@ -93,6 +150,28 @@ abstract class AppStyles {
...
+          );
+  static TextStyle normal14WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.button.copyWith(
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:173]: You should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 23]
```diff
@@ -136,6 +215,12 @@ abstract class AppStyles {
             color: Colors.white,
           );
 
+  static TextStyle normal18WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.bodyText1.copyWith(
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:221]: You should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 24]
```diff
@@ -145,19 +230,41 @@ abstract class AppStyles {
...
   static TextStyle normal20White(BuildContext context) =>
       Theme.of(context).textTheme.headline6.copyWith(
             fontWeight: FontWeight.normal,
             color: Colors.white,
           );
+  static TextStyle normal20WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.headline6.copyWith(
+            fontWeight: FontWeight.normal,
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:247]: You should use `color`

File: core/lib/common/themes/app_styles.dart [hunk 25]
```diff
@@ -145,19 +230,41 @@ abstract class AppStyles {
...
       Theme.of(context).textTheme.caption.copyWith(
             fontSize: 12,
             color: Colors.white,
           );
 
+  static TextStyle normal12WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.caption.copyWith(
+            fontSize: 12,
+            color: Colors.white,
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:265]: You should use color parameter here

File: core/lib/common/themes/app_styles.dart [hunk 26]
```diff
@@ -169,11 +276,26 @@ abstract class AppStyles {
...
+          );
+
   static TextStyle normal10Hint(BuildContext context) =>
       Theme.of(context).textTheme.overline.copyWith(
             color: AppColors.hintTextColor,
           );
 
+  static TextStyle normal10WithColor(BuildContext context,
+          {Color color = AppColors.white}) =>
+      Theme.of(context).textTheme.overline.copyWith(color: Colors.white);
```
Comment (AlexVegner) [inline core/lib/common/themes/app_styles.dart:292]: you should use color parameter here

**Review by AlexVegner — CHANGES_REQUESTED** [review]

**Comment by AlexVegner** [comment]
To many issues in PR. To risky for merge. 
PR closed.

**Pull request closed without merging**